gef.py

```diff
@@ -3899,16 +3899,12 @@ def get_memory_alignment(in_bits: bool = False) -> int:
 def clear_screen(tty: str = "") -> None:
     """Clear the screen."""
     global gef
-    if not tty:
-        gdb.execute("shell clear -x")
-        return
 
-    # Since the tty can be closed at any time, a PermissionError exception can
-    # occur when `clear_screen` is called. We handle this scenario properly
     try:
-        with open(tty, "wt") as f:
-            f.write("\x1b[H\x1b[J")
+        sys.stdout.write("\x1b[H\x1b[J")
```

Comment:
is `tty` even required 🤔 ?

Reply:
It doesn't seem like it is required, I guess we can just remove it from here